bucket/inkscape-extension-sozi-extras.json

```
@@ -0,0 +1,35 @@
+{
+    "version": "21.12.06",
+    "description": "Inkscape extension to insert video or audio to SVG document.",
+    "homepage": "https://sozi.baierouge.fr/",
+    "license": " MPL-2.0",
+    "depends": "inkscape",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/sozi-projects/Sozi/releases/download/v21.12.06/sozi-extras-media-21.12.06-1638805322-inkscape-1.0.zip",
+            "hash": "df9fa44dc4b52ea99745c2bf6dfd7583e1f2e47e51aeb1c11779e0398a97c815"
+        }
+    },
+    "post_install": [
+        "if (!(Test-Path \"$env:APPDATA\\inkscape\\extensions\")) {",
+        "   (New-Item -Type directory \"$env:APPDATA\\inkscape\\extensions\") | Out-Null",
+        "}",
+        "Copy-Item -Path \"$dir\\media-inkscape-1.0\" -Destination \"$env:APPDATA\\inkscape\\extensions\" -Recurse"
+    ],
+    "uninstaller": [
+        "if (Test-Path \"$env:APPDATA\\inkscape\\extensions\\media-inkscape-1.0\") {",
+        "   (Remove-Item \"$env:APPDATA\\inkscape\\extensions\\media-inkscape-1.0\")",
```

```suggestion
        "   Remove-Item \"$env:APPDATA\\inkscape\\extensions\\media-inkscape-1.0\" -Recurse",
```

Try using the Recurse flag in uninstaller

---

Still error, even with running powershell as admin

---

The appveyor also failed
```
[-] inkscape-extension-sozi-extras.json has 2 Errors!
      [*] Error: Invalid type. Expected Object but got Array.
        [^] Line: C:\projects\scoopinstaller-extras\bucket\inkscape-extension-sozi-extras.json:19:20
        [^] Path: http://scoop.sh/draft/schema#/definitions/uninstaller/Type
      [*] Error: JSON is valid against more than one schema from 'oneOf'. Valid schema indexes: 0, 1.
        [^] Line: C:\projects\scoopinstaller-extras\bucket\inkscape-extension-sozi-extras.json:19:20
        [^] Path: http://scoop.sh/draft/schema#/definitions/uninstaller/OneOf
      [-] inkscape-extension-sozi-extras.json 38ms
        Expected 0, but got 2.
        79:                         $validator.Errors.Count | should -be 0
        at <ScriptBlock>, C:\projects\scoop\test\Scoop-Manifest.Tests.ps1: line 79
```
It's not support multi line script `uninstaller` ?

---

Ah, got it. You need to declare a `script` field inside `uninstaller` object, and put this array inside that. Like this: https://github.com/ScoopInstaller/Extras/blob/59b802f3e6b9b9b52c3c632d449afb48f54f023c/bucket/paint.net-plugin-pyrochild.json#L14-L19